ui-v2/app/templates/dc/kv/index.hbs

```diff
@@ -18,6 +18,13 @@
             {{/if}}
         </h1>
     {{/block-slot}}
+    {{#block-slot 'toolbar'}}
+{{#if (gt items.length 0) }}
+      <form class="filter-bar">
+          {{freetext-filter onchange=(action 'filter') value=filter.s placeholder="Search by name"}}
+      </form>
+{{/if}}
```

Comment:
Do you maintain this style of indentation manually? Wondering how that might work if there's ever outside contributors / other front end devs on the project. (obviously not a concern right now, but was just curious!)

Reply:
Michael picked up on this also. Yeah, so basically the idea is there are essentially 2 levels of indentation happening. One for markup (or markup-like) code, and one for 'logic' code. Personally I think it helps to see what's happening in a template at a glance.

If I'm in 'logic-mode/js-mode' chances are I don't care about the markup at all, so mentally I can ignore the markup. If I'm in 'presentation-mode/css-html-mode' I can mentally ignore the logic stuff.

I'm not precious about it in the slightest tho